docs/source/AdministratorGuide/ServerInstallations/InstallingWebAppDIRAC.rst

```diff
@@ -256,154 +257,219 @@ Install and configure NGINX
 Note: you can run NGINX in a separate machine.
 
 The official site of NGINX is the following: `<http://nginx.org/>`_
-The required NGINX version has to be grater than 1.4.
+The required NGINX version has to be grater than 1.4 and WebDAV nginx module to serve static files.
+
+* Prepare, needed the development repository to compile the WebDAV dynamic module for Nginx::
 
-  * Install Nginx using package manager::
+      yum update -y
+      yum groupinstall "Development Tools" -y
+      yum install yum-utils pcre-devel zlib-devel libxslt-devel libxml2-devel -y
 
-         yum install nginx
+* Install Nginx using package manager. At this point, you should be able to install the pre-built Nginx package with dynamic module support::
 
+      yum install nginx -y
+      systemctl enable nginx
+      systemctl start nginx
 
 If your version is not grater than 1.4 you have to install NGINX manually.
```

Comment:
These types of instructions usually "decade" quickly. Can we just say that we support only Nginx 1.4+ ?

Reply:
Well agree, I don't think this line is really needed